recipes/spleeter/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "spleeter" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: fea6b1748e285cdef564d439f14500ed4a6c5ce8721f440e1dd7ae656749aaae
+  - url: https://pypi.io/packages/source/n/norbert/norbert-0.2.1.tar.gz
+    sha256: bd4cbc2527f0550b81bf4265c1a64b352cab7f71e4e3c823d30b71a7368de74e
+  - url: https://pypi.io/packages/source/m/musdb/musdb-0.3.1.tar.gz
+    sha256: 27aac71095f919c7e0cfb526af1e6f745275645e959f759c42711354ea0fcbf7
+  - url: https://pypi.io/packages/source/m/museval/museval-0.3.0.tar.gz
+    sha256: 2e1334dd3367dea562c1902515f6d55e87dc3072cfc411c616ac6f0a48d31d5f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - ffmpeg
+    - requests
+    - setuptools
+    - importlib_resources  # [py<37]
+    - tensorflow ==1.14.0
+    - pandas ==0.25.1
```

Comment:
Do these pins need to be this strict?

Reply:
Yes, this is a music processing library that performs source separation using deep learning. Dependencies has been frozen since our model has been learn using this version of Tensorflow and underlying deps could not been upgrade as well for compatibility purpose.